space/space.go

```diff
@@ -54,6 +55,7 @@ type Repository interface {
 	Load(ctx context.Context, ID satoriuuid.UUID) (*Space, error)
 	Delete(ctx context.Context, ID satoriuuid.UUID) error
 	List(ctx context.Context, start *int, length *int) ([]*Space, uint64, error)
```

Comment:
Do we still need List when we have Search?

I'm guessing List should be rewritten to take some form of 'account' context.
(different PR)